Tasks/AzureRmWebAppDeployment/make.json

```diff
@@ -32,5 +38,14 @@
             "source": "postDeploymentScript",
             "options": "-R"
         }
+    ],
+    "rm": [
```

Comment:
Multiple versions of vsts-task-lib affects the development of tasks. Should it be documented in contribution tab?
It would be better to take this to @ericsciple attention.